components/connector/cmd/main.go

```diff
@@ -43,10 +44,12 @@ type config struct {
 		Province           string `envconfig:"default=State"`
 	}
 	CertificateValidityTime     time.Duration `envconfig:"default=2160h"`
-	CASecretName                string        `envconfig:"default=namespace/name"`
+	CASecretName                string        `envconfig:"default=kyma-integration/connector-service-app-ca"`
 	RootCACertificateSecretName string        `envconfig:"optional"`
 
-	CertificateDataHeader string `envconfig:"default=Certificate-Data"`
+	CertificateDataHeader        string `envconfig:"default=Certificate-Data"`
+	RevocationConfigMapNamespace string `envconfig:"default=compass-system"`
+	RevocationConfigMapName      string `envconfig:"default=revocations-config"`
```

Comment:
I would suggest using one flag as `NamespacedName` as we do in case of `CASecretName` to be consistent,

Reply:
Done!